SuggestionRow.podspec

```diff
@@ -7,7 +7,7 @@ Pod::Spec.new do |s|
   s.author           = { "Xmartlabs SRL" => "swift@xmartlabs.com" }
   s.source           = { git: "https://github.com/EurekaCommunity/SuggestionRow.git", tag: s.version.to_s }
   s.social_media_url = 'https://twitter.com/EurekaCommunity'
-  s.ios.deployment_target = '8.0'
+  s.ios.deployment_target = '10.0'
```

Comment:
I would set this to iOS 9.3, the same as Eureka 5.

Reply:
Unless there is another reason to upgrade to iOS 10?

Reply:
@mats-claassen changes have been made. Also updated Carthage